dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -759,27 +759,21 @@ def predict(self, data, batch_size=32, show_confidences=False,
             allocation_index += num_samples_in_batch
 
         # Convert predictions, confidences to lists from numpy
-        predictions = predictions.tolist()
+        r_predictions = []
+        r_confidences = []
         if show_confidences:
-            confidences = confidences.tolist()
-        else:
-            confidences = None
-
-        # Trim array size to number of samples
-        if len(predictions) > allocation_index:
-            del predictions[allocation_index:]
-            if show_confidences:
-                del confidences[allocation_index:]
-
+            r_confidences = []
+                
         # Trim array size to length of sentence
```

Comment:
This comment seems out of date since you are no longer trimming the arrays but rather adding to them.